### DISCUSSION TITLE
Export and print features; user interface

### DISCUSSION THREAD

**Discussion by luc0x61** [nuttyartist/notes#518] [Ideas]
I'm testing the application right now, but one of the first things I feel need is an export path. I see it can export with its native format, to re-import and restore, that's fine.

However, how do I export to another format... namely, can I export a PDF of a group of notes? Print them? With this second option, I could anyway use a PDF printer and obtain my first goal.

--

The website recalls that the app can be docked and recalled with Command + N, but on Windows 11 that key combo is dedicated to recalling the notification bar ...

**Comment by bjorn** [comment]
> However, how do I export to another format... namely, can I export a PDF of a group of notes? Print them? With this second option, I could anyway use a PDF printer and obtain my first goal.

There is an open issue about exporting to text files (#227), but I think you're the first to bring up printing and PDF export. Definitely interesting features to add.

> The website recalls that the app can be docked and recalled with Command + N, but on Windows 11 that key combo is dedicated to recalling the notification bar ...

The website might be outdated. An updated documentation of the keyboard shortcuts is available here:

https://github.com/nuttyartist/notes/blob/master/docs/keyboard_shortcuts.md

On Windows 11, due to the conflict, this shortcut has been changed to <kbd>Win</kbd> + <kbd>Shift</kbd> + <kbd>N</kbd>.

**Reply by luc0x61** [reply]
@bjorn with _this shortcut has been changed to_ do you mean _will be changed to_ ...? Because right now it seems to not be assigned automatically to Win + Shift + N, i.e. it has no effect in my Win 11 testing, either with the window active or not

**Reply by bjorn** [reply]
I meant the shortcut has been changed in Notes 2.1, so if you have that installed, `Win` + `Shift` + `N` should work.

**Reply by luc0x61** [reply]
Oh well, then I have an issue with that. Didn't work on Win 11 pro, and it doesn't work on another machine with Win 10 pro 22H2 where I installed it right now

**Reply by zjeffer** [reply]
@luc0x61 Can you create an issue for this? It should work fine on Windows 11 as far as I know.

**Comment by nuttyartist** [comment]
Thanks for the heads up! I'll update the website with the correct shortcut key.

Exporting to PDF sounds like a cool idea. I'd love to see how other apps are doing that and take inspiration. I guess there are many ways one can export the Markdown to PDF. There are probably many libraries out there that do it already and we just need to wrap it with our UI. I'll create a new issue for that.
